pxr/usd/usd/crateFile.h

```diff
@@ -79,13 +79,36 @@ template <class T> struct _IsBitwiseReadWrite;
 
 enum class TypeEnum : int32_t;
 
-// Value in file representation.  Consists of a 2 bytes of type information
-// (type enum value, array bit, and inlined-value bit) and 6 bytes of data.
-// If possible, we attempt to store certain values directly in the local
-// data, such as ints, floats, enums, and special-case values of other types
-// (zero vectors, identity matrices, etc).  For values that aren't stored
-// inline, the 6 data bytes are the offset from the start of the file to the
-// value's location.
+/* usddoc value_rep
+ * Crate stores Value Representations as data blobs that are read on demand.
+ * This allows large USD scenes to be read incredibly quickly by deferring all
+ * data reads to the latest point possible.
+ *
+ * Value reps are stored as an unsigned 64 bit integer.
```

Comment:
```suggestion
 * Value representations are stored as an unsigned 64 bit integer.
```
prefer full words to aid full text indexing and search